### DISCUSSION TITLE
GitHub mobile turn off GitHub URL grab

### DISCUSSION THREAD

**Discussion by keskival** [community/community#45711] [Mobile]
Since GitHub mobile doesn't support almost any of the useful features of the GitHub desktop browser, like copy repository clone URL, or a million other things, would it be possible to have a settings switch for not grabbing the GitHub URL links I click on my mobile browser, which take me to the useless app, and don't let me back to the browser?
Would it also be possible to have a button in the app for "take me to the web browser"?

**Comment by jge162** [comment]
Best option is to uninstall the app until that becomes a feature.

**Reply by dan33play4** [reply]
$ pip install errbot
$ mkdir errbot; cd errbot
$ errbot --init
$ errbot

**Reply by robertjustice** [reply]
FroggyDogBountyhunter2
Looks great

**Comment by robertjustice** [comment]
Mikey Pertee

**Reply by includingshaikh-hub** [reply]
World like to me about the my self feedback

**Comment by robertjustice** [comment]
Mikey Pertee got the code of conduct

**Reply by robertjustice** [reply]
Robbie1orjust26689@gmail.com
